index.html

```diff
@@ -2040,6 +2044,13 @@ <h3>Evidence</h3>
     "evidenceDocument": "DriversLicense",
     "subjectPresence": "Physical",
     "documentPresence": "Physical"
+  },{
+    "id": "https://example.edu/evidence/f2aeec97-fc0d-42bf-8ca7-0548192dxyzab",
```

Comment:
Hmm, don't quite understand this. I'm going to merge anyway and try to rework the examples by putting in a "LoA" example, and a physical document example.